src/environments/README.md

```diff
@@ -1,15 +1,138 @@
-# Cloud environments
+# ArcCtl Environments
 
-## Example
+Environments represent running instances of [components](../components/) that that
+are made available to each other for integration. You likely have environments for
+things like `production` or `staging`, but may also create private environments for
+integration testing or everyday development.
 
-```yaml
-components:
-  architect/frontend:
-    source: architect/frontend:latest
+ArcCtl attempts to automate as much configuration for components as possible, but
+there are still cases where owners will want to assign environment-specific rules
+and values.
+
+## Creating environments
+
+Before you can create an environment, you'll need a [datacenter](../datacenters/) the
+environment can be backed by. To learn more about datacenters, check out the [datacenters docs](../datacenters/).
+
+### Ephemeral environments
```

Comment:
I think the up command is a little bit different because it's bound to the process itself. If I wanted to setup a preview/feature environment on (for example) every pull request, I'd actually have to use `arcctl create env` instead of `arcctl up`. The fact that it MUST live with the process is why I called it ephemeral.

Also, since we're shifting gears and focusing a bit more on operations, ephemeral is a fine word to use. We'll definitely be creating other guides at a later date that show someone how to create preview or feature environments on pull requests, but that's not really what the up command is for.